ompi/mca/osc/ucx/osc_ucx_component.c

```diff
@@ -825,7 +825,9 @@ int ompi_osc_ucx_free(struct ompi_win_t *win) {
         ucp_worker_progress(mca_osc_ucx_component.ucp_worker);
     }
 
-    opal_common_ucx_worker_flush(mca_osc_ucx_component.ucp_worker);
+    if (OMPI_SUCCESS != opal_common_ucx_worker_flush(mca_osc_ucx_component.ucp_worker)) {
+        OSC_UCX_VERBOSE(1, "opal_common_ucx_worker_flush failed");
```

Comment:
pls print return code in all 3 error messages

Reply:
added error codes